app/Services/Post/PostService.php

```diff
@@ -37,7 +36,24 @@ public function getCategories()
      */
     public function store($request)
     {
-        return $this->postDao->store($request);
+        $image = $request->file('file');
+        if (is_null($image)) {
+            $imageName = 'default.png';
+        } else {
+            $imageName = $image->getClientOriginalName();
+            $location = 'images';
+            $image->move($location, $imageName);
+        }
+
+        $data = (object) [
+            'title' => $request->title,
+            'description' => $request->description,
+            'status' => $request->status,
+            'categories' => $request->categories,
+            'image' => $imageName
+        ];
+
+        return $this->postDao->store($data);
```

Comment:
**Remove**
```
 $data = (object) [
            'title' => $request->title,
            'description' => $request->description,
            'status' => $request->status,
            'categories' => $request->categories,
            'image' => $imageName
        ];
```

Add

`$request->image = $imageName;`
`return $this->postDao->store($request);`